### ISSUE TITLE
False Possitve on decorator code - Too many positional arguments for function callpylint(too-many-function-args)

### ISSUE THREAD

**Issue by aernesto24** [PyCQA/pylint#3172]
I am using this code:
```
# -*- coding: utf-8 -*-
def protected(func):

    def wrapper(password):

        if password == 'er12345':
            return func()
        else:
            print('La contraseña es incorrecta!!!')
    
    return wrapper


@protected
def protected_func():
    print('Tu contraseña es correcta')


if __name__ == '__main__':
    password = str(raw_input('Ingresa tu contraseña: '))

    protected_func(password)
```

but protected_func(password) in the list lime appears as an error:
```
Too many positional arguments for function callpylint(too-many-function-args)
```

the solution is compiling and working as expected but it is marking the last line as an error.

I am using Visual studio code Version 1.35.1 (1.35.1) 
python 2.7.0 64 bits
Extension Python 2019.6.24221

**Comment by aernesto24** [comment]
MacOS

**Comment by PCManticore** [comment]
Thanks for opening an issue. `pylint` does not understand decorators that mutate their wrapped function so that those functions accept an additional number of arguments. In pylint 2.4 you can ignore these kind of decorators using the new option `--signature-mutators=` as in `--signature-mutators=protected`
